src/core/Akka/Event/EventBus.cs

```diff
@@ -225,5 +225,10 @@ private List<TSubscriber> UpdateCacheForEventClassifier(TEvent @event, TClassifi
                 return list;
             }
         }
+
+        public bool IsSubscribing(TClassifier eventClass)
```

Comment:
So this checks to see if anyone is subscribed to a given topic? Can you rename this method to `HasSubscribers` ?

Reply:
Also, you will probably need to approve this change to the public API here via: https://getakka.net/community/public-api-changes.html